File: BP_Front/BP/src/app/Interfaces/MovimientoDTO.ts
export interface MovimientoDTO {
    idMovimiento: number;
    idCuenta: number;
    numeroCuenta: string;   
    fecha: Date;
    tipoMovimiento: number;
    tipoMovimientoDescripcion: string;
    valor: number;
    saldo: number;
    clienteNombre: string;
  }
  

File: BP_API/BP_API/DTO/MovimientoDTO.cs
namespace BP_API.DTO
{
    public class MovimientoDTO
    {
        public int IdMovimiento { get; set; }
        public int IdCuenta { get; set; }
        public string? NumeroCuenta { get; set; }
        public DateTime? Fecha { get; set; }
        public int TipoMovimiento { get; set; }
        public string? TipoMovimientoDescripcion { get; set; } = string.Empty;
        public decimal Valor { get; set; }
        public decimal? Saldo { get; set; }
        public string? ClienteNombre { get; set; } = string.Empty;
    }
}